MELA/python/mela_binding.cpp

```diff
@@ -215,6 +217,29 @@ SimpleParticleCollection_t collection_initializer(py::list listOfParticles){
     return collection;
 }
 
+void setInputEvent(Mela& mela, SimpleParticleCollection_t* daughters, SimpleParticleCollection_t* associated, SimpleParticleCollection_t* mothers, bool isgen){
+    if((daughters) && (daughters->size() == 0)){
+        daughters = 0;
+    }
+    if ((associated) && (associated->size() == 0)){
+        associated = 0;
+    }
+    if ((mothers) && (mothers->size() == 0)){
+        mothers = 0;
+    }
+    mela.setInputEvent(daughters, associated, mothers, isgen);
+}
+
+void PrintCurrentCandidateSummary(Mela& mela){
+    MELACandidate* curCand = mela.getCurrentCandidate();
+    TUtil::PrintCandidateSummary(curCand);
+}
+
+void PrintCandidateSummary(Mela& mela, TVar::simple_event_record curCand){
```

Comment:
I don't think a Mela object needs to be passed here.

Reply:
Pertaining to the comment on L233, you could make another version that receives MELACandidate. You would be overloading the function, so make sure the type is disambiguated correctly.